blog/content/post/2020-05-14-qubesOs.md

```diff
@@ -0,0 +1,129 @@
+---
+title: Qubes Os and 3mdeb 'minisummit' 2020
+abstract: 'Once again, we will meet on QubesOs & 3mdeb minisummit 2020 discussing #QubesOS,
+ #firmware, #coreboot, #security and #TPM related topics. All the event details are presented in the following blog post.'
+cover: /covers/qubes&3mdeb_logo.png
+author: kamila.banecka
+layout: post
+published: true
+date: 2020-05-14
+archives: "2020"
+
+tags:
+  - qubesOs
+  - minisummit
+  - live
+  - youTube
+  - firmware
+  - coreboot
+  - security
+  - TPM
+categories:
+  - Miscellaneous
+  - Security
+  - Firmware
+
+
+---
+As we wrote [here](https://blog.3mdeb.com/2019/2019-08-07-qubes-os-and-3mdeb-minisummit/), recent Qubes OS 'minisummit' 2019 turned out to be fruitful and productive
+event. Great piece of knowledge has been presented and many ideas came up from
+long and deep discussions. Positively built with this experience, as observing
+changes inside security of firmare and computer architecture, we  couldn’t miss
+the opportunity and decided to organize Qubes OS minisummit, edition 2020. We
+want  to share our knowledge and our experience around topics presented below
+hoping, that once more, we go far beyond the agenda. This year, we will stand
+side by side with Frédéric Pierret and Marek Marczykowski Górecki from QubesOS
+team, covering the following topics:
+
+* 20.05
+  1. Michał Żygowski Qubes OS on modern AMD platform
+  2. Norbert Kamiński Status fwupd/LVFS support for Qubes OS
+* 28.05
+  1. Piotr Król SRTM for Qubes OS VMs
+  2. Michał Żygowski Anti Evil Maid for modern AMD UEFI-based platform
+* 04.06
+  1. Piotr Król DRTM for Qubes OS VMs
+  2. Michał Żygowski Anti Evil Maid for Intel coreboot-based platform
+* 10.06
+  1. Frédéric Pierret: How to build Qubes? From components to operating system
+overview.
+  2. Marek Marczykowski-Górecki Operating system testing, when it itself uses
+ virtual machines.
+
+
+Each stand will be divided into broad status report, design talk with proposals
+and discussion ended with  **Q&A** time for our viewers, so they could actively
+participate in the dialogue. We hope that the most promising presentations will
+be transformed into community supported or publicly founded projects.
+
+## Where to link the event?
+
+We are going to stream transmission via OBS on our YouTube 3mdeb channel. All
+you need to do is joining our [live](https://www.youtube.com/channel/UC_djHbyjuJvhVjfT18nyqmQ/live) in an appropriate time: all data details you will find below. Minisummit starts
```

Comment:
@KBanecka wrap text after the link - link can take more than 80 chars rule, but there shouldn't be any text in this line after the url